R/1.0_module_data_summary.R

```diff
@@ -196,7 +196,7 @@ get_object_filter_overview_array <- function(filtered_data, unfiltered_data, dat
   if (length(subject_keys) == 0) {
     data.frame(
       dataname = dataname,
-      obs = nrow(unfiltered_data),
+      obs = ifelse(!is.null(nrow(unfiltered_data)), nrow(unfiltered_data), NA),
```

Comment:
@averissimo why only for `obs` and not for `obs_filtered`

Reply:
@m7pr This is because some datasets might be added in the transform, so they don't have `_raw`

Reply:
got it! totally makes sense now! thanks for the reply